pandas_log/patched_logs_functions.py

```diff
@@ -31,17 +43,32 @@
 
 # TIPS
 ITERROWS_TIPS = "\t*iterrows is not recommended, and in the majority of cases will have better alternatives"
-FILLNA_NO_NA_TIP = "\t* There are no nulls in this dataframe, if you are working on the entire dataset you can remove this operation."
-SHOULD_REDUCED_ROW_TIP = "\t* Number of rows didn't change, if you are working on the entire dataset you can remove this operation."
+FILLNA_NO_NA_TIP = (
+    "\t* There are no nulls in this dataframe, if you are working on the entire dataset you can "
+    "remove this operation."
+)
+SHOULD_REDUCED_ROW_TIP = (
+    "\t* Number of rows didn't change, if you are working on the entire dataset you can remove "
+    "this operation."
+)
 
 # Others
-SORT_VALUES_MSG = "\t* Sorting by columns {by} in a {'ascending' if ascending else 'descending'} order."
+SORT_VALUES_MSG = "\t* Sorting by columns {} in a {} order."
 SORT_INDEX_MSG = "\t* Sorting by index in a {'ascending' if ascending else 'descending'} order."
 DEFAULT_STRATEGY_USED_MSG = (
     "\t* Using default strategy (some metric might not be relevant)."
 )
 TRANSFORMED_TO_DF_MSG = "\t* After transformation we received Series"
 
+COPY_WARNING_MSG = (
+    "Some pandas logging may involve copying dataframes, which can be time-/memory-intensive. "
+    "Consider passing copy_ok=False to the enable/auto_enable functions in pandas_log if issues arise."
+)
+
+
+def _stringify_list(l):
```

Comment:
The functions further down that do something like eg ",".join(some_column_names) break if the column names are integers rather than strings. I use this to wrap the join argument in those cases, eg ','.join(_stringify_list(some_column_names)). Just seemed cleaner than writing out the same list comprehension each time.